docs/modules/tiered-storage/pages/configuration.adoc

```diff
@@ -0,0 +1,179 @@
+= Configuring Tiered Storage
+
+== Before you Begin
+
+Tiered Storage works with maps whose in-memory format is `NATIVE`. Therefore, you need to enable this format
+before you start configuring Tiered Storage. You can enable it using the declarative configuration; add the following
+to your Hazelcast configuration file.
+
```

Comment:
Maybe we can add a link somewhere that explains how to configure native memory? We don't explain the other configuration here.

Reply:
For example, is `persistent-memory` needed as well? I would just link to the page about HD memory instead of providing this example config. It's more confusing and it doesn't offer the Java examples either.